card_pools/misc.json

```diff
@@ -0,0 +1,12 @@
+[
```

Comment:
i'm not sure these two make sense as card pools as is.

Draft would really be draft + anything, likely.

NAPD multiplayer would be the same.  

maybe just best to leave them out for now until we address draft deck building head on?